modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -503,9 +550,21 @@ public Assignment getAssignment(User.ID userID, Application.Name applicationName
                 experimentBatch.getLabels().remove(experiment.getLabel());
             }
         }
+        if(LOGGER.isDebugEnabled()) LOGGER.debug("allAssignments: {} ", allAssignments);
```

Comment:
Added isDebugEnabled as having isDebugEnabled avoids a string concatenation in the case if logging LEVEL is not debug..